xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-sources/src/main/java/org/xwiki/notifications/sources/internal/UsersParameterHandler.java

```diff
@@ -90,6 +102,24 @@ public void handleUsersParameter(String users, NotificationParameters parameters
             parameters.filters.add(new FollowedUserOnlyEventFilter(entityReferenceSerializer, userList));
 
             addFilterPreference(parameters, userList);
+        } else if (parameters.user != null) {
+            // if we have a user (but no "users") then we should also display personal messages from followed users.
+            // the other types of messages get included, but for personal messages the filter needs
+            // a matching filter preference so we loop though preferences to see if they have
+            // a preference for this (using a copy to guard against unwanted modifications)
```

Comment:
Minor comment, sentences begin with a capital letter and end with a period.

Reply:
Thanks, fixed.